### PR TITLE
ARXIVNG-2400 added developers page to /about

### PR DESCRIPTION
**Pull request by erickpeirson**
Adds a page at ``/about/people/developers`` and attendant links.

image

### CODE REVIEW

File: about/people/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ For more information about who is arXiv please see:
 
 - [Staff Leadership Team](leadership_team)
 - [Moderators](/moderators/)
+- [Developers](developers)
```
Comment (jimentwood) [inline about/people/index.md]: How about Developers -> Volunteer Developers

Comment (erickpeirson) [reply about/people/index.md]: Fixed in 1b6c7f5

File: about/people/developers.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# Volunteer Developers
+
+The technical infrastructure and software behind arXiv.org is operated and
+maintained by a core team of dedicated IT staff at Cornell. However, it would
+not be possible to keep pace with the ever-changing landscape of internet
+technology, nor the evolving needs of 21st century science, without the help of
+dedicated arXiv users who contribute to the open source software that powers
+the arXiv platform.
+
+We are grateful to the following external developers for their contributions to
```
Comment (jlmorano) [inline about/people/developers.md]: Do you want to use "volunteer" in place of "external" developers?

Comment (erickpeirson) [reply about/people/developers.md]: Yes, good idea; fixed in b7880fe

**Pull request merged**